darts/ad/scorers/kmeans_scorer.py

```diff
@@ -0,0 +1,216 @@
+        KMeans model for each dimension. If the input series has a dimension of D, the model will train D KMeans
+        models. If set to False, the model will concatenate the dimension in the considered `window` W and compute
+        the score using only one trained KMeans model.
+
+        Training with ``fit()``:
+
+        The input can be a series (univariate or multivariate) or a sequence of series. The series will be partitioned
+        into equal size subsequences. The subsequence will be of size W * D, with:
+            - W being the size of the window given as a parameter `window` (w>0)
+            - D being the dimension of the series (D=1 if deterministic)
+
+        For a series of length N, (N-W+1)/W subsequences will be generated. If a list of series is given of length L,
+        each series will be partitioned into subsequences, and the results will be concatenated into an array of
+        length L * number of subsequences.
+
+        The model KMeans will be fitted on the generated subsequences. The model will find `k` cluster of dimensions
+        equal to the length of the subsequences (D*W).
+
+        If `component_wise` is set to True, the algorithm will be applied to each dimension independently. For each
+        dimension, a KMeans model will be trained.
+
+        Compute score with ``score()``:
+
+        The input can be a series (univariate or multivariate) or a sequence of series. The given series must have the
+        same dimension D as the data used to train the KMeans model.
+
+        - If the series is multivariate of dimension D:
+            - if `component_wise` is set to False: it will return a univariate series (dimension=1). It represents
+            the anomaly score of the entire series in the considered window at each timestamp.
+            - if `component_wise` is set to True: it will return a multivariate series of dimension D. Each dimension
+            represents the anomaly score of the corresponding dimension of the input.
+
+        - If the series is univariate, it will return a univariate series regardless of the parameter
+        `component_wise`.
+
+        A window of size W is rolled on the series with a stride equal to 1. It is the same size window W used during
+        the training phase. At each timestamp, the previous W values will form a vector of size W * D
+        of the series (with D being the series dimensions). The KMeans model will then retrieve the closest centroid
+        to this vector and compute the euclidean distance between the centroid and the vector. The output will be a
+        series of dimension one and length N-W+1, with N being the length of the input series. Each value will represent
+        how anomalous the sample of the W previous values is.
+
+        If a list is given, a for loop will iterate through the list, and the function ``_score_core()`` will be
```

Comment:
That's too many details for the users (they don't need to know there's a `_score_core()` function etc).